src/main/java/com/collreach/userprofile/service/impl/UserProfileServiceImpl.java

```diff
@@ -75,7 +75,7 @@ public String checkUsername(UserSignupRequest userSignupRequest){
     @Override
     public String checkEmail(UserSignupRequest userSignupRequest){
         Boolean email = userPersonalInfoRepository.existsByEmail(userSignupRequest.getEmail());
-        Boolean altEmail = userPersonalInfoRepository.existsByAlternateEmail(userSignupRequest.getAlternateEmail());
+        Boolean altEmail = userPersonalInfoRepository.existsByAlternateEmail(userSignupRequest.getEmail());
```

Comment:
This change will take the **email** from the request only _**not**_ _alternate email_ whether we have to check **_alternateEmail_** or the **_email_**.